src/julia.h

```diff
@@ -1520,7 +1520,7 @@ typedef struct _jl_task_t {
 } jl_task_t;
 
 JL_DLLEXPORT jl_task_t *jl_new_task(jl_function_t *start, size_t ssize);
-JL_DLLEXPORT void jl_switchto(jl_task_t *t);
+JL_DLLEXPORT void jl_switchto(jl_task_t **pt);
```

Comment:
Not complaining, just observing mostly. I guess with 1.0 we should consider having a stable C API. Either that or say that it's not stable and that language SemVer doesn't apply to internals.